CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ BUG FIXES:
  * core: Fixed bug in reconciler where allocs already stopped were being unnecessarily updated [[GH-4764](https://github.com/hashicorp/nomad/issues/4764)]
  * client: Fix an issue reloading the client config [[GH-4730](https://github.com/hashicorp/nomad/issues/4730)]
  * driver/raw_exec: Fix issue where tasks that used an interpolated command in driver configuration would not start [[GH-4813](https://github.com/hashicorp/nomad/pull/4813)]
+ * core: Fix an issue with validating jobs with artifact checksums containing meta variables and interpolation [[GH-4810](https://github.com/hashicorp/nomad/pull/4819)]
```

Comment:
wording suggestion - `Fix an issue where artifact checksums containing interpolated variables failed validation` to make it more inline with the style used in other entries.